src/app/app.component.html

```diff
@@ -166,6 +170,13 @@
               [searchInputField]="searchInputField"
               [currentFolder]="selectedFolder"
               matSuffix matTooltip="Show search options"></app-search-expression-builder> -->
+          <button *ngIf="searchText!==''"
+            mat-icon-button
+            matSuffix
+              (click)="showSaveSearchDialog()"
+              matTooltip="Save search">
+            <mat-icon>save</mat-icon>
```

Comment:
svgIcons? gonna be a mess if we keep both kinds..

Reply:
Fixed in 3cef8ae, along with the other examples that sneaked in.